examples/01_opening_floris_computing_power.py

```diff
@@ -33,46 +33,56 @@
 fi = FlorisInterface("inputs/gch.yaml")
 
 # Convert to a simple two turbine layout
-fi.reinitialize(layout_x=[0, 500.], layout_y=[0., 0.])
+fi.reinitialize(layout_x=[0, 500.0], layout_y=[0.0, 0.0])
 
 # Single wind speed and wind direction
-print('\n========================= Single Wind Direction and Wind Speed =========================')
+print("\n========================= Single Wind Direction and Wind Speed =========================")
 
-# Get the turbine powers assuming 1 wind speed and 1 wind direction
-fi.reinitialize(wind_directions=[270.], wind_speeds=[8.0])
+# Get the turbine powers assuming 1 wind direction and speed
+fi.reinitialize(wind_directions=[270.0], wind_speeds=[8.0])
 
 # Set the yaw angles to 0
-yaw_angles = np.zeros([1,1,2]) # 1 wind direction, 1 wind speed, 2 turbines
+yaw_angles = np.zeros([1, 2])  # 1 wind direction / speed, 2 turbines
 fi.calculate_wake(yaw_angles=yaw_angles)
 
 # Get the turbine powers
-turbine_powers = fi.get_turbine_powers()/1000.
-print('The turbine power matrix should be of dimensions 1 WD X 1 WS X 2 Turbines')
+turbine_powers = fi.get_turbine_powers() / 1000.0
+
+# TODO what should we call this user/facing?
```

Comment:
Have you resolved this?

Reply:
I was thinking to leave it as FINDEX until/unless we decide on something else?

Reply:
Agreed, I think that's OK for now